components/tel/Core/Inc/radio.h

```diff
@@ -1,48 +1,56 @@
-
 /**
  *  @file radio.h
  *  @brief header file for radio.c. Define buffer lengths
+ * 
+ *  Read this monday item for architecture details: https://ubcsolar26.monday.com/boards/7524367653/pulses/7524368294
  *
- *  @date 2023/03/18
+ *  @date 2024/10/10
  *  @author Aarjav Jain
  */
 
+
```

Comment:
I don't think adding data and author makes sense since these files will be changed and modified in the future by different people. The real data the author is tracked by git.